implementations/rust/ockam/ockam_app/src/enroll/enroll_user.rs

```diff
@@ -60,7 +66,8 @@ async fn enroll_with_token(app_state: &AppState) -> Result<IdentityIdentifier> {
 
 async fn retrieve_space(app_state: &AppState) -> Result<Space> {
     info!("retrieving the user space");
-    let node_manager = app_state.node_manager.get().read().await;
+    let node_manager_worker = app_state.node_manager().await;
```

Comment:
Maybe we should rename `app_state.node_manager()` to `app_state.node_manager_worker()`. 
I was going the ambiguity at first but now that both are more exposed it's probably better to be more explicit.

Reply:
Done!